google/cloud/spanner_dbapi/version.py

```diff
@@ -16,8 +16,14 @@
 
 def google_client_info(user_agent=None):
     """
-    Return a google.api_core.gapic_v1.client_info.ClientInfo
-    containg the user_agent and python_version for this library
+    Return a :class:`google.api_core.gapic_v1.client_info.ClientInfo`
```

Comment:
Better put it into plain words instead of using a formal class label. Also, this method is refactored out in PR [#532](https://github.com/googleapis/python-spanner-django/pull/532).